.github/workflows/federated-integ-test.yml

```diff
@@ -0,0 +1,201 @@
+---
+
+name: Federated integ test
+
+#on:  # yamllint disable-line rule:truthy
+#  workflow_dispatch:
+#  workflow_run:
+#    workflows:
+#      - build-all
+#    branches:
+#      - master
+#      - 'v1.*'
+#    types:
+#      - completed
+#  push:
+#    branches:
+#      - master
+#      - 'v1.*'
+#  pull_request:
+#    branches:
+#      - master
+#      - 'v1.*'
+#    types: [opened, reopened, synchronize]
```

Comment:
this is commented out to avoid this to be run
This will be fixed  by 
https://github.com/magma/magma/pull/13012